testbed/tests/trace_test.go

```diff
@@ -126,6 +135,52 @@ func TestTraceNoBackend10kSPSJaeger(t *testing.T) {
 	}
 }
 
+func TestTraceNoBackend10kSPSZipkin(t *testing.T) {
```

Comment:
This is a duplicate of TestTraceNoBackend10kSPSJaeger. Instead of duplicating code it is better to add another looping dimension to TestTraceNoBackend10kSPSJaeger, so that it performs the same scenarios using different receivers/exporters (and rename TestTraceNoBackend10kSPSJaeger to TestTraceNoBackend10kSPS).

Reply:
@tigrannajaryan so the question is, should I leave this tests as it was (because it needs also specific config yaml file with defined Zipkin receiver) or add as test case zipkin receiver and create two new config yamls?